medicines/web/tools/sitemap-generator.ts

```diff
@@ -11,7 +10,7 @@ const BASE_URL = 'https://products.mhra.gov.uk';
 const YYY_MM_DD = 'YYYY-MM-DD';
 const CHANGE_FREQUENCY = 'daily';
 
-const createPathsObj = async () => {
+const createPathsObj = async (): Promise<{ [index: string]: any }> => {
```

Comment:
Can we change the value of any to `string`

```ts
 Promise<{ [index: string]: string }
```

Reply:
@pataruco But the value is an object, not a string.

```
{
  '/about': { page: '/about', lastModified: '2020-01-07T13:24:17.208Z' },
  '/accessibility': { page: '/accessibility', lastModified: '2020-01-07T13:24:17.208Z' },
  '/cookies': { page: '/cookies', lastModified: '2020-01-07T13:24:17.208Z' },
  '/': { page: '/', lastModified: '2020-01-07T13:24:17.208Z' }
}
```

Is there a way to specify a data structure like this? Is that worthwhile?

Reply:
Cool, leave it like that 👍